Apps/Sandcastle/gallery/Particle System Weather.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport"
+          content="width=device-width, initial-scale=1, maximum-scale=1, minimum-scale=1, user-scalable=no">
+    <meta name="description" content="Particle systems for rain and snow.">
+    <meta name="cesium-sandcastle-labels" content="Beginner, Showcases, Tutorials">
+    <title>Cesium Demo</title>
+    <script type="text/javascript" src="../Sandcastle-header.js"></script>
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+        require.config({
+            baseUrl : '../../../Source',
+            waitSeconds : 60
+        });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+
+    #toolbar {
+        background: rgba(42, 42, 42, 0.8);
+        padding: 4px;
+        border-radius: 4px;
+    }
+
+    #toolbar input {
+        vertical-align: middle;
+        padding-top: 2px;
+        padding-bottom: 2px;
+    }
+
+    #toolbar .header {
+        font-weight: bold;
+    }
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar"></div>
+<script id="cesium_sandcastle_script">
+    function startup(Cesium) {
+        'use strict';
+//Sandcastle_Begin
+/********* Set up the viewer, terrain, and camera *********/
+var viewer = new Cesium.Viewer('cesiumContainer', {
+    shouldAnimate : true
+});
+viewer.terrainProvider = new Cesium.CesiumTerrainProvider({
+    url : 'https://assets.agi.com/stk-terrain/v1/tilesets/world/tiles'
+});
+
+var scene = viewer.scene;
+scene.globe.depthTestAgainstTerrain = true;
+scene.camera.setView({
+    destination : new Cesium.Cartesian3(277096.634865404, 5647834.481964232, 2985563.7039122293),
+    orientation : {
+        heading : 4.731089976107251,
+        pitch : -0.32003481981370063
+    }
+});
+
+// temp variables for math
+var gravityScratch = new Cesium.Cartesian3();
+
+// snow
+var snowOptions = {
+    minParticleSize : scene.drawingBufferWidth / 60.0,
+    radius : 100000.0,
+
+    update : function(particle, dt) {
+        var position = particle.position;
+
+        Cesium.Cartesian3.normalize(position, gravityScratch);
+
+        var magnitude = -5500.0;
+        Cesium.Cartesian3.multiplyByScalar(gravityScratch, magnitude, gravityScratch);
+
+        particle.velocity = Cesium.Cartesian3.add(particle.velocity, gravityScratch, particle.velocity);
```

Comment:
The snow should fall a little slower I think. Also to mimic a little air resistance, can we add some randomness to the velocity as it falls?